plugins/lookup/dig.py

```diff
@@ -95,6 +96,21 @@
     msg: "MX record for gmail.com {{ item }}"
   with_items: "{{ lookup('community.general.dig', 'gmail.com./MX', wantlist=true) }}"
 
+- name: lookup multiple names at once
+  ansible.builtin.debug:
+    msg: "A record found {{ item }}"
+  loop: "{{'community.general.dig', 'example.org.', 'example.com.', 'gmail.com.', wantlist=true) }}"
```

Comment:
```suggestion
  loop: "{{ lookup('community.general.dig', 'example.org.', 'example.com.', 'gmail.com.', wantlist=true) }}"
```
Alternatively use `query`, then `wantlist=true` isn't needed:
```suggestion
  loop: "{{ query('community.general.dig', 'example.org.', 'example.com.', 'gmail.com.') }}"
```

Reply:
Done.